codersdk/deployment.go

```diff
@@ -1265,6 +1306,29 @@ when required by your organization's security policy.`,
 			Group:         &deploymentGroupConfig,
 			Value:         &c.Config,
 		},
+		{
+			Name:        "CLI SSH Hostname Prefix",
+			Description: "The CLI SSH deployment name is the used in the Hostname of the ssh config.",
+			Flag:        "cli-ssh-hostname-prefix",
+			Env:         "SSH_HOSTNAME_PREFIX",
+			YAML:        "sshHostnamePrefix",
+			Group:       &deploymentGroupClient,
+			Value:       &c.SSHConfig.DeploymentName,
+			Hidden:      false,
+			Default:     "coder.",
+		},
+		{
+			Name: "SSH Config Options",
+			Description: "These ssh config options will override the default ssh config options. " +
+				"Provide options in key=value format separated by commas." +
+				"Using this incorrectly can break ssh to your deployment. Use cautiously.",
```

Comment:
```suggestion
			Description: "These SSH config options will override the default SSH config options. " +
				"Provide options in key=value format separated by commas. " +
				"Using this incorrectly can break SSH for your deployment, use cautiously.",
```

Let's stylize SSH the same way in all places.

Should we mention space separation is also possible? They behave slightly differently.

Reply:
Do they behave differently? I guess I can include it. If you do this with flags/env you just need to quote your values.

Reply:
There's a little difference, not sure there's any benefit to the whitespace variant though:

> Arguments may optionally be enclosed in double quotes (") in order to represent arguments containing spaces.  Configuration options may be separated by whitespace or optional whitespace and exactly one ‘=’; the latter format is useful to avoid the need to quote whitespace when specifying configuration options